2022-02-14 23:11:54,694 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from /N/u/soodn/Carbonate/.config/wandb/settings
2022-02-14 23:11:54,695 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/settings
2022-02-14 23:11:54,695 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-14 23:11:54,696 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'models/5-Deepflash2/Deepflash2-Train-Colon.py', 'program': '/geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/Deepflash2-Train-Colon.py'}
2022-02-14 23:11:54,696 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 23:11:54,696 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 23:11:54,696 INFO    MainThread:36798 [wandb_init.py:_log_setup():386] Logging user logs to /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/run-20220214_231154-3inylxtk/logs/debug.log
2022-02-14 23:11:54,697 INFO    MainThread:36798 [wandb_init.py:_log_setup():387] Logging internal logs to /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/run-20220214_231154-3inylxtk/logs/debug-internal.log
2022-02-14 23:11:54,697 INFO    MainThread:36798 [wandb_init.py:init():420] calling init triggers
2022-02-14 23:11:54,697 INFO    MainThread:36798 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-02-14 23:11:54,697 INFO    MainThread:36798 [wandb_init.py:init():471] starting backend
2022-02-14 23:11:54,698 INFO    MainThread:36798 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-14 23:11:54,793 INFO    MainThread:36798 [backend.py:ensure_launched():219] starting backend process...
2022-02-14 23:11:54,959 INFO    MainThread:36798 [backend.py:ensure_launched():224] started backend process with pid: 37254
2022-02-14 23:11:54,983 INFO    MainThread:36798 [wandb_init.py:init():480] backend started and connected
2022-02-14 23:11:55,000 INFO    MainThread:36798 [wandb_init.py:init():550] updated telemetry
2022-02-14 23:11:55,461 INFO    MainThread:36798 [wandb_init.py:init():581] communicating current version
2022-02-14 23:11:59,291 INFO    MainThread:36798 [wandb_init.py:init():586] got version response 
2022-02-14 23:11:59,292 INFO    MainThread:36798 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-14 23:11:59,384 INFO    MainThread:36798 [wandb_init.py:init():624] starting run threads in backend
2022-02-14 23:12:04,396 INFO    MainThread:36798 [wandb_run.py:_console_start():1827] atexit reg
2022-02-14 23:12:04,402 INFO    MainThread:36798 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-02-14 23:12:04,403 INFO    MainThread:36798 [wandb_run.py:_redirect():1706] Redirecting console.
2022-02-14 23:12:04,404 INFO    MainThread:36798 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-14 23:12:04,404 INFO    MainThread:36798 [wandb_init.py:init():651] run started, returning control to user process
2022-02-14 23:12:09,276 INFO    MainThread:36798 [wandb_run.py:_config_callback():966] config_cb None None {'Learner': {'loss_func': 'JointLoss(\n  (first): WeightedLoss(\n    (loss): TorchLoss(\n      (loss): DiceLoss()\n    )\n  )\n  (second): WeightedLoss()\n)', 'opt_func': 'fastai.optimizer.ranger', 'lr': 0.001, 'splitter': 'fastai.torch_core.trainable_params', 'metrics': ['<hubmap_loss_metrics.Dice object at 0x2b9a500cd310>', '<hubmap_loss_metrics.Iou object at 0x2b9a500cd4c0>', '<hubmap_loss_metrics.Recall object at 0x2b9a500cd520>', '<hubmap_loss_metrics.Precision object at 0x2b9a500cd580>'], 'path': '.', 'model_dir': 'models', 'wd': 0.0001, 'wd_bn_bias': False, 'train_bn': True, 'moms': [0.95, 0.85, 0.95], '_name': '<fastai.learner.Learner object at 0x2b9a522b0490>'}, 'TrainEvalCallback': True, 'Recorder': {'add_time': True, 'train_metrics': False, 'valid_metrics': True}, 'ProgressCallback': True, 'SaveModelCallback': {'fname': 'model', 'every_epoch': False, 'at_end': False, 'with_opt': False}, 'ElasticDeformCallback': True, 'WandbCallback': {'log': 'gradients', 'log_preds': False, 'log_model': False, 'log_dataset': False, 'dataset_name': None, 'valid_dl': None, 'n_preds': 36, 'seed': 12345, 'reorder': True}, 'MixedPrecision': True, 'ParamScheduler': True, 'n_inp': 1, 'input 1 dim 1': 32, 'input 1 dim 2': 3, 'input 1 dim 3': 512, 'input 1 dim 4': 512, 'batch size': 32, 'batch per epoch': 16, 'model parameters': 10046436, 'device': 'cuda', 'frozen': False, 'frozen idx': 0}
2022-02-14 23:12:09,281 INFO    MainThread:36798 [wandb_watch.py:watch():43] Watching
2022-02-14 23:22:34,114 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from /N/u/soodn/Carbonate/.config/wandb/settings
2022-02-14 23:22:34,120 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/settings
2022-02-14 23:22:34,121 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-14 23:22:34,122 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'models/5-Deepflash2/Deepflash2-Train-Colon.py', 'program': '/geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/Deepflash2-Train-Colon.py'}
2022-02-14 23:22:34,123 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 23:22:34,124 INFO    MainThread:36798 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-14 23:22:34,125 INFO    MainThread:36798 [wandb_init.py:_log_setup():386] Logging user logs to /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/run-20220214_232234-3uzsxm35/logs/debug.log
2022-02-14 23:22:34,125 INFO    MainThread:36798 [wandb_init.py:_log_setup():387] Logging internal logs to /geode2/home/u090/soodn/Carbonate/ccf-research-kaggle-2021/models/5-Deepflash2/wandb/run-20220214_232234-3uzsxm35/logs/debug-internal.log
2022-02-14 23:22:34,126 INFO    MainThread:36798 [wandb_init.py:init():420] calling init triggers
2022-02-14 23:22:34,128 INFO    MainThread:36798 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-02-14 23:22:34,129 INFO    MainThread:36798 [wandb_init.py:init():441] re-initializing run, found existing run on stack: 3inylxtk
2022-02-14 23:22:34,130 INFO    MainThread:36798 [wandb_run.py:finish():1468] finishing run soodn/bricknet/3inylxtk
2022-02-14 23:22:34,131 INFO    MainThread:36798 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-14 23:22:34,137 INFO    MainThread:36798 [wandb_run.py:_restore():1769] restore
2022-02-14 23:22:36,556 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 904
}

2022-02-14 23:22:36,723 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 904
}

2022-02-14 23:22:36,921 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 904
  total_bytes: 19367
}

2022-02-14 23:22:37,024 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3141
  total_bytes: 19367
}

2022-02-14 23:22:37,127 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19367
  total_bytes: 19367
}

2022-02-14 23:22:37,232 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19367
  total_bytes: 19367
}

2022-02-14 23:22:37,336 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19367
  total_bytes: 19367
}

2022-02-14 23:22:37,440 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19367
  total_bytes: 19367
}

2022-02-14 23:22:37,751 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19367
  total_bytes: 19367
}

2022-02-14 23:22:37,954 INFO    MainThread:36798 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 19367
  total_bytes: 19367
}
local_info {
}

2022-02-14 23:22:39,552 INFO    MainThread:36798 [wandb_run.py:_append_history():2144] rendering history
2022-02-14 23:22:39,555 INFO    MainThread:36798 [wandb_run.py:_append_summary():2102] rendering summary
2022-02-14 23:22:39,556 INFO    MainThread:36798 [wandb_run.py:_append_files():2194] logging synced files
